src/constants/constants.js

```diff
@@ -0,0 +1,5 @@
+export const SUPPORTED_FILE_EXTENSIONS = {
+  CSV: 'csv',
+};
```

Comment:
Maybe it's better to convert it into array? `SUPPORTED_FILE_EXTENSIONS = ['csv']`.